site/reference/live/preview/README.md

```diff
@@ -9,8 +9,27 @@ description: >
     Preview the changes apply would make to the cluster
 -->
 
-`preview` shows the operations that will be done when running `apply` or 
-`destroy` against a cluster.
+`preview` validates the resources in the package against the cluster and shows
+which resources will be applied and pruned.
+
+If the `preview` command is used without specifying the `server-side` flag,
+validation and dry-run will primarily happen client-side. `kpt` will display the 
+operations that will be performed against the apiserver for each resource when
+running `apply`. The operations are:
+ * `created`: The resource doesn't currently exist in the cluster and will be created.
+ * `configured`: The resource exists in the cluster and will be updated or remain unchanged.
+ * `failed`: The resource can't be applied.
```

Comment:
How about other categories: {pruned, skipped}?

Also why "failed" appears twice?

```
6 resource(s) applied. 0 created, 0 unchanged, 6 configured, 0 failed (preview)
0 resource(s) pruned, 0 skipped, 0 failed (preview)
```

Which of these are mutually exclusive?